### DISCUSSION TITLE
What is CB benchmark?

### DISCUSSION THREAD

**Discussion by zhimin-z** [open-compass/opencompass#616] [Q&A] [answered]
https://opencompass.org.cn/dataset-detail/CB

**Comment by yingfhu** [comment, marked as answer]
You could refer config here https://github.com/open-compass/opencompass/blob/main/configs/datasets/SuperGLUE_CB/SuperGLUE_CB_gen_854c6c.py

or here https://huggingface.co/datasets/super_glue/viewer/cb
